Panic in fabric-peer-ext when running fabric unit tests

- FAIL: TestVersionedDB_UpdateCache (0.01s)
    --- FAIL: TestVersionedDB_UpdateCache/SaveCacheUpdates (0.00s)
        statecouchdb_ext_test.go:177: 
            	Error Trace:	statecouchdb_ext_test.go:177
            	Error:      	func (assert.PanicTestFunc)(0x11c7d60) should not panic
            	            		Panic value:	runtime error: invalid memory address or nil pointer dereference
            	            		Panic stack:	goroutine 1009 [running]:
            	            	runtime/debug.Stack(0xc000303cf6, 0x12d5700, 0x1e6d940)
            	            		/opt/hostedtoolcache/go/1.14.4/x64/src/runtime/debug/stack.go:24 +0xab
            	            	github.com/stretchr/testify/assert.didPanic.func1.1(0xc000303d28, 0xc000303cf6, 0xc000303d18)
            	            		/tmp/tmp.atpRGxOxSq/pkg/mod/github.com/stretchr/testify@v1.6.1/assert/assertions.go:1013 +0xc0
            	            	panic(0x12d5700, 0x1e6d940)
            	            		/opt/hostedtoolcache/go/1.14.4/x64/src/runtime/panic.go:975 +0x3e3
            	            	github.com/trustbloc/fabric-peer-ext/pkg/gossip/state.(*UpdateHandler).getCacheUpdater(...)
            	            		/tmp/tmp.atpRGxOxSq/src/github.com/hyperledger/fabric/fabric-peer-ext/pkg/gossip/state/stateupdate.go:142
            	            	github.com/trustbloc/fabric-peer-ext/pkg/gossip/state.SaveCacheUpdates(0x0, 0x0, 0x3e9, 0xc0001c2000, 0x13, 0x13)
            	            		/tmp/tmp.atpRGxOxSq/src/github.com/hyperledger/fabric/fabric-peer-ext/pkg/gossip/state/cacheupdater.go:34 +0x181
            	            	github.com/hyperledger/fabric/extensions/gossip/state.SaveCacheUpdates(...)

--- a/core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb_ext_test.go
+++ b/core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb_ext_test.go
@@ -160,8 +160,7 @@ func TestVersionedDB_UpdateCache(t *testing.T) {
 	)
 
 	vdb := &VersionedDB{
-		cache:             newCache(10, nil),
-		storeCacheUpdates: true,
+		cache: newCache(10, nil),
 	}
 
 	t.Run("UpdateCache", func(t *testing.T) {

